packages/service-library/src/web-api/api-contracts/privacy-scan-types/privacy-result-response.ts

```diff
@@ -0,0 +1,13 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+import { DeepScanResultItem, ScanReport, ScanRun } from '../scan-result-response';
+
+export type privacyResultResponse = {
```

Comment:
no need for this type, we will use existing structure with `privacyScan?: boolean;` flag